src/redux/greetings/greetings.js

```diff
@@ -0,0 +1,25 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const API_URL = 'http://localhost:3000/api/v1/greetings';
+
+const FETCH = 'hello-rails-react/greetings/FETCH';
+
+const initialState = {
+  greeting: '',
+};
+
+export const fetchGreeting = createAsyncThunk(FETCH, async () => {
+  const response = await axios.get(API_URL);
+  const { data } = response;
+  return data.message;
+});
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case `${FETCH}/fulfilled`:
+      return { ...state, greeting: action.payload };
+    default:
+      return state;
+  }
+};
```

Comment:
- [ ] I am not able to fetch `greetings` on the frontend setup `cors` it might be the issue
<img width="1429" alt="Screen Shot 2022-12-16 at 22 23 14" src="https://user-images.githubusercontent.com/44745855/208183111-9f0e4c6b-5624-4c9f-b983-685c87e9ddd9.png">

Reply:
Hi @DeliceLydia 

It is working on my side but I don't know why it is not running on your side.

Are you running the server?